### ISSUE TITLE
Incorrect value for 

### ISSUE THREAD

**Issue by jane-t** [amaximus/anniversary#13]
I have an event as follows

```
- platform: anniversary
  name: events
  multiple: true
  items: 3
  anniversaries:
    - event: 'Jeannette'
      date: '1970-01-02'
      icon: mdi:cake-variant
      type: birthday

```
However the value for the age at next birthday is currently showing as 51, when it should be 52.   I suspect this might be to do with the fact the birthday is in the following year, as the other two birthday ages which are in December are showing correctly.

```
first_event_name: Christa
first_event_in: 15
first_event_on: '2021-12-25'
anniversary: 76
events:
  - event: Christa
    event_in: '15'
    event_on: '2021-12-25'
    anniversary: '76'
    type: birthday
    icon: mdi:cake-variant
  - event: Stephen
    event_in: '16'
    event_on: '2021-12-26'
    anniversary: '60'
    type: birthday
    icon: mdi:cake-variant
  - event: Jeannette
    event_in: '23'
    event_on: '2022-01-02'
    anniversary: '51'
    type: birthday
    icon: mdi:cake-variant
friendly_name: events
icon: mdi:cake-variant
```

**Comment by amaximus** [comment]
Just puhed a fix (version 0.2.1)

Please update the integration, test and get back to me with results.

**Comment by jane-t** [comment]
![2021-12-10_141253](https://user-images.githubusercontent.com/920792/145587331-7b3e3ae8-d18e-48bd-aff8-27d9d53a1b9e.jpg)

That looks fine.  Thank you for fixing it so quickly.
